src/archutils/Win32/RegistryAccess.cpp

```diff
@@ -62,11 +62,8 @@ OpenRegKey(const std::string& sKey, RegKeyMode mode, bool bWarnOnError = true)
 							   &hRetKey);
 	if (retval != ERROR_SUCCESS) {
 		if (bWarnOnError)
-			LOG->Warn(
-			  werr_ssprintf(
-				retval, "RegOpenKeyEx(%x,%s) error", hType, sSubkey.c_str())
-				.c_str());
-		return nullptr;
+			Locator::getLogger()->warn(werr_ssprintf(retval, "RegOpenKeyEx(%x,%s) error", hType, sSubkey.c_str()));
+		return NULL;
```

Comment:
This seems unintentional.

Reply:
I'll fix that, it probably got changed when doing the rebase, it was a pretty long rebase, and a few things like this and with includes did get through.